### PR TITLE
Beta 2 Release Note Improvements

### PR DESCRIPTION
**Pull request by nadinequinr3**
Draft preview: https://nadine.preview.docs.r3.com/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.html

- Removed some pieces, as requested.
- Elaborated and clarified some changes.
- Added a div to fix the width of the first column, instead of spaces.
- Used new groupings in the first column.
- Used new rows instead of bullets for each change.
- Moved the mTLS tutorial out of "Onboarding" and added a link to the configuration fields section.

(Ledger API changes still require more info. Will be addressed in a future PR.)

### CODE REVIEW

**Review by yarivus — APPROVED** [review]
Lgtm

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ section_menu: corda-5-beta
...
 |                                              | **Corda CLI** - Any worker and CLI long options that used camel case now use kebab case. This may require that existing scripts are updated. | 
 |                                              | **Messaging Maximum Size** - A new `messaging` [configuration field]({{< relref "../operating/configuration/config-overview.md" >}}), `maxAllowedMessageSize`, enables you to specify the maximum size of a message sent from a Corda worker to the message bus. Corda breaks messages that exceed this size into smaller messages before sending. |                                           
-| **CorDapp Development**                      | **Java API** - The Kotlin API has been replaced with a Java API. Kotlin and Java developers can now use this to build CorDapps. Existing CorDapps must be written.|
+| **CorDapp Development**                      | **Java API** - The Kotlin API has been replaced with a Java API. As a result, existing CorDapps must be written. Kotlin and Java developers can now use this API to build CorDapps. |
```
Comment (yarivus) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md:21]:
```suggestion
| **CorDapp Development**                      | **Java API** - The Kotlin API has been replaced with a Java API. As a result, existing CorDapps must be rewritten. Kotlin and Java developers can now use this API to build CorDapps. |
```
Or ...
```suggestion
| **CorDapp Development**                      | **Java API** - The Kotlin API has been replaced with a Java API. As a result, existing CorDapps must be adapted. Kotlin and Java developers can now use this API to build CorDapps. |
```

**Review by yarivus — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md [hunk 2]
```diff
@@ -9,35 +9,34 @@ menu:
...
+|                                              | **Operational Statuses** - Four new operational statuses have been added for virtual nodes to provide fine-grained control over its flows and what components they can interact with: <li>`flowOperationalStatus` — describes a virtual node's ability to start new flows. </li> <li>`flowP2pOperationalStatus` — describes a virtual node's ability to communicate with peers.</li> <li>`flowStartOperationalStatus`  — describes a virtual node's ability to run flows, to have checkpoints, and to continue in-progress flows. </li> <li> `vaultDbOperationalStatus` —  describes a virtual node's ability to perform persistence operations on the virtual node's vault. </li> These four statuses replace the existing `state` field. The `GET virtualnode` now returns the status of each of the four operational characteristics. Maintenance mode has been updated to disable all four of these new operational statuses. You can set this mode using the existing change virtual node state endpoint: `/api/v1/virtualnode/{virtualNodeShortHash}/state/{newState}`.
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0-beta/release-notes/release-notes-c5-2.0.md:30]:
```suggestion
|                                              | **Operational Statuses** - Four new operational statuses have been added for virtual nodes to provide fine-grained control over their flows and what components they can interact with: <li>`flowOperationalStatus` — describes a virtual node's ability to start new flows. </li> <li>`flowP2pOperationalStatus` — describes a virtual node's ability to communicate with peers.</li> <li>`flowStartOperationalStatus`  — describes a virtual node's ability to run flows, to have checkpoints, and to continue in-progress flows. </li> <li> `vaultDbOperationalStatus` —  describes a virtual node's ability to perform persistence operations on the virtual node's vault. </li> These four statuses replace the existing `state` field. The `GET virtualnode` now returns the status of each of the four operational characteristics. Maintenance mode has been updated to disable all four of these new operational statuses. You can set this mode using the existing change virtual node state endpoint: `/api/v1/virtualnode/{virtualNodeShortHash}/state/{newState}`. 
```

**Review by AnnaK-R3 — APPROVED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by colin-hillis — CHANGES_REQUESTED** [review]

**Comment by colin-hillis** [comment]
I have tried a few approaches to merging cells for the first column, but nothing has worked so far. This is perhaps something that we can circle back on soon, but for now, there are bigger priorities.

**Pull request merged**